Update docs to remove usage of the word "anomaly." (#2340)
spacetelescope/romancal: docs/roman/wfi18_transient/description.rst

This algorithm requires a minimum of 5 resultants to be successful.  If fewer resultants
are present, or if the fit fails for any reason, the step will fall back on simply

> masking the rows most affected by the transient signal.  In this case, the first

Reviewer: ```suggestion
masking the rows most affected by the transient.  In this case, the first
```

----------

The WFI18 transient correction algorithm is based on work by T. Brandt,

> "Characterizing and Correcting the First Read Transient Signal of Detector 18 on Roman-WFI",

Reviewer: ```suggestion
"Characterizing and Correcting a High Amplitude Transient in Detector 18 on Roman-WFI",
```
